packages/framework/src/Model/Payment/SpaydHelper.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Payment;
+
+class SpaydHelper
+{
+    /**
+     * @param string $iban   IBAN účtu příjemce (např. CZ5508000000001535784133)
+     * @param float|null $amount Částka (např. 1250.50), může být null
+     * @param string $currency Měna, např. CZK nebo EUR
+     * @param string|null $bic  BIC/SWIFT (povinné pro zahraniční platbu v EUR)
+     * @param string|null $vs   Variabilní symbol
+     * @param string|null $ss   Specifický symbol
```

Comment:
please avoid the shortcuts for the parameter names. Also, remove/translate the comments please